lib/shared/generate-circuit-json.tsx

```diff
@@ -48,15 +48,18 @@ export async function generateCircuitJson({
   const runner = new userLandTscircuit.RootCircuit({
     platform: platformConfig,
   })
-  const projectDir = path.dirname(filePath)
-  const resolvedOutputDir = outputDir || projectDir
+  const absoluteFilePath = path.isAbsolute(filePath)
```

Comment:
```suggestion
  const resolvedFilePath = path.isAbsolute(filePath)
```

This can be absolute or relative right?

Reply:
yes
<img width="637" height="531" alt="image" src="https://github.com/user-attachments/assets/892904c6-df60-45f3-995c-467b2f441241" />
<img width="615" height="231" alt="image" src="https://github.com/user-attachments/assets/1e98ba66-cbec-40a4-b8c8-8c9680480967" />